src/components/AllProjects/index.jsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import allProjectsQuery from './graphql/allProjectsQuery';
+import Request from "../utilities/Request";
+import Project from './components/Project';
+import './AllProjects.css';
+
+class AllProjects extends React.Component {
+  state = {
+    projects: this.props.data.projects
```

Comment:
initialize as `null` since cDM sets it

Reply:
if i initialize as null, it disrupts the renderProjects fn.
![image](https://user-images.githubusercontent.com/29721784/44827406-dbd7e180-abc7-11e8-9691-9dc00366fc7e.png)